packages/core/test/unit/dialects/mssql/connection-manager.test.ts

```diff
@@ -0,0 +1,122 @@
+import type { Options } from '@sequelize/core';
+import { ConnectionError, Sequelize } from '@sequelize/core';
+import type { MsSqlDialect } from '@sequelize/mssql';
+import type { RequiredBy } from '@sequelize/utils';
+import { assert, expect } from 'chai';
+import sinon from 'sinon';
+import { Connection as TediousConnection } from 'tedious';
+import { getTestDialect } from '../../../support';
+
+const dialect = getTestDialect();
+
+type TestConnection = Omit<TediousConnection, 'once' | 'removeListener' | 'on'> & {
+  once(event: string, cb: () => void): void;
+  removeListener(): void;
+  on(): void;
+};
+
+if (dialect === 'mssql') {
```

Comment:
prefer having this if inside the describe to reduce nesting:

```ts
describe(..., () => {
  if (dialect !== 'mssql') {
    return;
  }
})
```

Reply:
Done